### PR TITLE
Remove label from talk search input

### PR DESCRIPTION
**Pull request by marcoambrosini**
**Before** 

Screenshot from 2023-11-17 08-07-45

**After**

Screenshot from 2023-11-17 08-06-04

### CODE REVIEW

**Comment by marcoambrosini** [comment]
@ShGKme @Antreesy do you know why the placeholder is not showing unless the input is focused?

**Comment by DorraJaouad** [comment]
> @ShGKme @Antreesy do you know why the placeholder is not showing unless the input is focused?

Missing `label-outside` ?

**Comment by marcoambrosini** [comment]
Also removed the white outline

Screenshot from 2023-11-17 08-18-35

File: src/components/LeftSidebar/SearchBox/SearchBox.vue
Comment (Antreesy) [inline src/components/LeftSidebar/SearchBox/SearchBox.vue:174]:
```suggestion
		&:focus,
		&:focus-visible,
		&:focus-within,
		&:hover {
```
Not sure if needed, but might worth a check with focus on trailing button and keyboard navigation

**Comment by nickvergessen** [comment]
I thought the label was required for a11y, that's also why we have it on the login page?

**Comment by marcoambrosini** [comment]
>I thought the label was required for a11y, that's also why we have it on the login page?

Not needed with search icon and aria label https://ergebnis.bitvtest.de/pruefschritt/bitv-20-web/bitv-20-web-9-3-3-2-beschriftungen-von-formularelementen-vorhanden

**Comment by marcoambrosini** [comment]
@michaelnissenbaum could you confirm that it's ok to use aria-label when we provide this type of context?

File: src/components/LeftSidebar/SearchBox/SearchBox.vue
Comment (ShGKme) [reply src/components/LeftSidebar/SearchBox/SearchBox.vue:174]: `focus-within` is not needed, there are no elements "within" `<input>`.

For `focus-visible` we have only `unset` for browser styles. When input is on focus, it has `:focus` state anyway.

**Comment by Antreesy** [comment]
Rebased onto master
- added `label-outside` prop to always show placeholder
- as input height is 44px now, remove paddings to align component with buttons in TopBar

**Review by Antreesy — APPROVED** [review]
Tested. Waiting for another review

**Review by DorraJaouad — APPROVED** [review]

**Comment by michaelnissenbaum** [comment]
@marcoambrosini Hi, if the magnifying glass icon meets the contrast requirements and the aria-label attribute is present, then everything is fine.

**Pull request merged**
